cmd/eks-node-viewer/main.go

```diff
@@ -184,6 +184,7 @@ func startMonitor(ctx context.Context, settings *monitorSettings) {
 						log.Println("unable to find node", n.Name)
 					} else {
 						node.Update(n)
+						node.UpdatePrice(settings.pricing)
```

Comment:
I kept getting nodes printing without pricing data if I scaled-up while the viewer was running. It looks like the labels aren't there on `Create` anymore when Karpenter uses machines, so I added `UpdatePrice` on `Update` and scaled to 200 nodes, seems to work well.